src/Symfony/Component/Filesystem/Tests/FilesystemTest.php

```diff
@@ -783,6 +791,10 @@ public function testChgrpLink()
 
     public function testChgrpSymlinkFails()
```

Comment:
just out of curiosity, there is no `@require unix` annotation or sth like that?

Reply:
Indeed: https://docs.phpunit.de/en/10.5/annotations.html#requires

But across the code base, it seems we use more `if ('\\' === \DIRECTORY_SEPARATOR)`. I guess this way allows to be more precise by adding a description on why it is skipped?